(sync) Resolve conflicts with gatsby-i18n-source @ 79b09bc (#172)
gatsbyjs/gatsby-ja: docs/tutorial/building-a-theme.md

2. The `Event` type will implement the typical Gatsby `Node` interface.
3. You'll use `@dontInfer`, because rather than Gatsby inferring fields, you'll be defining them explicitly.
4. In addition to an `id` field, you'll create new fields for each data point associated with an event (name, location, startDate, endDate, url). _To read more detail about creating types, check out the [`createTypes` documentation](/docs/actions/#createTypes)_.

> 5. You'll also create a `slug` field. You'll notice your event data doesn't include "スラッグ" data. You'll define this in the next step.

Reviewer: ```suggestion
5. You'll also create a `slug` field. You'll notice your event data doesn't include "slug" data. You'll define this in the next step.
```

> ### Define resolvers for any custom fields (スラッグ）

Reviewer: ```suggestion
### Define resolvers for any custom fields (slug）
```

4. In addition to an `id` field, you'll create new fields for each data point associated with an event (name, location, startDate, endDate, url). _To read more detail about creating types, check out the [`createTypes` documentation](/docs/actions/#createTypes)_.
5. You'll also create a `slug` field. You'll notice your event data doesn't include "slug" data. You'll define this in the next step.

> ### Define resolvers for any custom fields (slug）

Reviewer: ```suggestion
### Define resolvers for any custom fields (slug)
```

}
```

> You'll define a helper, `slugify` to help generate the スラッグ s:

Reviewer: ```suggestion
You'll define a helper, `slugify` to help generate the slugs:
```

yarn workspace gatsby-theme-events develop
```

> If you query this time for `allEvent`, you'll see the `Event` data, including the new スラッグ s:

Reviewer: ```suggestion
If you query this time for `allEvent`, you'll see the `Event` data, including the new slugs:
```
